tools/proto-js-plugin/src/main/java/io/spine/js/gradle/Extension.java

```diff
@@ -58,6 +69,32 @@
      * The absolute path to the test Protobufs compiled to JavaScript.
      */
     public String testGenProtoDir;
+    /**
+     * Names of JavaScript modules and Protobuf packages they provide.
+     *
+     * <p>Information about modules is used to resolve imports in generated Protobuf files.
+     *
+     * <p>Predefined Spine Modules are {@linkplain #resolveSpineModules included} by default.
```

Comment:
The link should be not here, but in a place, which describes how to override the default behavior.